cinema/views.py

```diff
@@ -21,7 +25,7 @@
     MovieSessionDetailSerializer,
     MovieListSerializer,
     OrderSerializer,
-    OrderListSerializer,
+    OrderListSerializer, MovieImageSerializer,
```

Comment:
Please, correct)

Reply:
Make these imports in a column  - one on each line